CONTRIBUTING.md

```diff
@@ -53,9 +53,8 @@ There are several channels where you can get answers from the community
 or the maintainers of this project:
 
 - Our public forum, [Kong Nation](https://discuss.konghq.com), is great for
-  asking questions, giving advice, and staying up-to-date with the latest
-  announcements. Kong Nation is frequented by Kong maintainers.
-- Two chat channels are used by the community, but are rarely visited by Kong
+  asking questions, giving advice, and staying up-to-date with the latest announcements. Kong Nation is frequented by Kong maintainers.
```

Comment:
What is the purpose of this change? The newline was on purpose, as to no exceed our limit of 80 columns per line.